src/keyword-mark.ts

```diff
@@ -44,6 +44,21 @@ export class KeywordMarkElement extends HTMLElement {
     this.setAttribute('keywords', val);
   }
 
+  /**
+   * Delimiter for keywords
+   */
+  public get delimiter(): string {
+    return this.getAttribute('delimiter') || '';
```

Comment:
should probably be `??` here

Reply:
Yeah! Ive updated the keywords attribute with this too.